sources/ClangSharp.PInvokeGenerator/Abstractions/FunctionOrDelegateDesc.cs

```diff
@@ -16,83 +16,83 @@ public bool IsVirtual
         {
             get => (Flags & FunctionOrDelegateFlags.IsVirtual) != 0;
             set => Flags =
-                value ? Flags | FunctionOrDelegateFlags.IsVirtual : Flags ^ FunctionOrDelegateFlags.IsVirtual;
+                value ? Flags | FunctionOrDelegateFlags.IsVirtual : Flags & FunctionOrDelegateFlags.IsVirtual;
```

Comment:
The false case needs to be `Flags & ~FunctionOrDelegateFlags.IsVirtual` so that it preserves all bits except for the one being cleared.

Reply:
Yeah I realised that, but left it on the college PC and forgot to push! Will fix tomorrow.